hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml

```diff
@@ -175,6 +175,12 @@
       <groupId>org.jline</groupId>
       <artifactId>jline</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.blinkfox</groupId>
+      <artifactId>mini-table</artifactId>
+      <version>1.0.0</version>
```

Comment:
This version needs to be added in the parent project, We need to make sure there are no CVE issues.

Reply:
Thanks for the advice. I'll check it out